cap/modules/records/jsonschemas/definitions/file-v1.0.0.json

```diff
@@ -1,39 +1,37 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "properties": {
-    "_files": {
-      "items": {
-        "description": "Describe information needed for files in records.",
-        "properties": {
-          "bucket": {
-            "type": "string"
-          },
-          "checksum": {
-            "type": "string"
-          },
-          "key": {
-            "type": "string"
-          },
-          "previewer": {
-            "description": "Identifier for previewer needed to preview this file.",
-            "type": "string"
-          },
-          "size": {
-            "type": "integer"
-          },
-          "type": {
-            "description": "File type extension.",
-            "type": "string"
-          },
-          "version_id": {
-            "type": "string"
-          }
-        },
-        "type": "object"
-      },
-      "type": "array"
-    }
-  },
-  "title": "CAP Files Schema v1.0.0",
-  "type": "object"
+ "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
This schema is a "base" schema to be used with records and deposits, and is connected with Invenio_v3 functionalities ( invenio-records, invenio-deposit, invenio-files, etc ).

We can either 
(1) change it back to the array structure with "_files" and if needed can be also used somewhere else
(2) change it back and create a new sub-schema that is of the same manner as the one amended